src/views/Map/Map.tsx

```
@@ -127,29 +127,29 @@ export function MapPage() {
       <div className={s.mapWrap}>
         <Suspense fallback={null}>
           <ConnectedMap className={s.Map} />
-          {featureFlags[FeatureFlag.ADMIN_BOUNDARY_BREADCRUMBS] && (
+          {featureFlags[AppFeature.ADMIN_BOUNDARY_BREADCRUMBS] && (
             <Plus16 className={s.crosshair}></Plus16>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)